internal/setup/setup.go

```diff
@@ -47,7 +47,7 @@ type Package struct {
 type Slice struct {
 	Package   string
 	Name      string
-	Essential []SliceKey
+	Essential map[SliceKey]bool
```

Comment:
This has changed since the last review, and it doesn't look natural.  The de-duplication is a simple local need inside the particular algorithm that is parsing the slices.  It doesn't justify changing how the whole code sees the list of essentials.

Reply:
I have changed it back. Instead of creating a map I am iterating over the list of essentials now. I assume the list will not be long enough to justify hashmap vs iteration over a slice, and the map requires extra allocations.